distribution/pom.xml

```
@@ -130,6 +130,38 @@
                                     </arguments>
                                 </configuration>
                             </execution>
+                            <execution>
+                                <id>generate-dependency-reports</id>
+                                <phase>initialize</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${project.parent.basedir}/docs/_bin/generate-license-dependency-reports.py</executable>
+                                    <arguments>
+                                        <argument>${project.basedir}/../</argument>
+                                        <argument>${project.basedir}/target</argument>
+                                        <argument>--parallel</argument>
+                                        <argument>4</argument>
```

We could also determine this in Python:
```suggestion
import multiprocessing
multiprocessing.cpu_count()
```

---

I didn't use that API intentionally because I think it's too intense. Please note that this command is executed whenever you build the distribution.